docs/blocks/feature-flags-and-experimental-interfaces.md

```diff
@@ -0,0 +1,53 @@
+We have feature gating system setup in our plugin that defines what is accessible to the public and what is not, it has three phases:
+
+*   **Core flag `WOOCOMMERCE_BLOCKS_PHASE=1`:** anything that is not hidden behind a flag, falls under this category, and it contains all of the code that runs on WooCommerce Core plugin.
+*   **Feature plugin flag** `**WOOCOMMERCE_BLOCKS_PHASE=**2`**:** anything that is behind this flag is code that is shipped to our [feature plugin](https://wordpress.org/plugins/woo-gutenberg-products-block/), the files of blocks behind this flag are also present in WooCommerce Core, just not active.
+*   **Experimental flag** `**WOOCOMMERCE_BLOCKS_PHASE**=3`: This flag contains things that we're not shipping yet, so unfinished work mostly. These features are only available in developer builds of the plugin.
```

Comment:
There is an issue with bolds in these lines. They got inside the code expression:
```suggestion
*   **Experimental flag** `WOOCOMMERCE_BLOCKS_PHASE=3`: This flag contains things that we're not shipping yet, so unfinished work mostly. These features are only available in developer builds of the plugin.
```

Reply:
Good catch, I used an html to markdown tool and missed this bad transformation.